docs/products/opensearch/howto/connect-with-python.rst

```diff
@@ -0,0 +1,44 @@
+Connect to OpenSearch cluster with Python
+=========================================
+
+You can interact with your cluster with the help of the `Python OpenSearch client <https://github.com/opensearch-project/opensearch-py>`_. 
+It provides a convenient syntax to send commands to your OpenSearch cluster. Follow its ``README`` file for installation instructions.
+
+To connect with your cluster, you need the **Service URI** of your OpenSearch cluster. You can find the connection details in the `**Overview** <https://console.aiven.io/project/dev-sandbox/services>`_ page. Alternatively, you can find it in Aiven command line interface `service command <https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get>`_. ``service_uri`` contains credentials, therefore should be treated with care.
```

Comment:
One more minor things - I think for readers who don't know our cli well, would be helpful to see which command they need. Now the link to cli brings to correct place, but the description is still wage. What is we use
```suggestion
To connect with your cluster, you need the **Service URI** of your OpenSearch cluster. You can find the connection details in the `**Overview** <https://console.aiven.io/project/dev-sandbox/services>`_ page. Alternatively, you can retrieve it by running command `service get <https://developer.aiven.io/docs/tools/cli/service.html#avn-service-get>`_ from Aiven command line interface.  ``service_uri`` contains credentials, therefore should be treated with care.
```

Reply:
I brought the changes and changed to use ref.